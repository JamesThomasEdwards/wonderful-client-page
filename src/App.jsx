// React;
import React, { Component } from 'react';
// Files;
import TopMiddleShapeContainer from './containers/TopMiddleShapeContainer.jsx';
import LeftShapeContainer from './containers/LeftShapeContainer.jsx';
import AboutProfileCardContainer from './containers/AboutProfileCardContainer.jsx';
import BottomLeftShapeContainer from './containers/BottomLeftShapeContainer.jsx';

export default class App extends Component {
    state = {};
    render() {
        return (
            < div>
                <TopMiddleShapeContainer />
                <AboutProfileCardContainer />
                <LeftShapeContainer />
                <BottomLeftShapeContainer />
            </div >
        );
    };
};
