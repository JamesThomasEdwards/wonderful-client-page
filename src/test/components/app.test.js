// React;
import React from 'react';
// Files;
import App from '../../App.jsx';
// Testing
import ReactShallowRenderer from 'react-test-renderer/shallow';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

test('should render App component correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<App />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});