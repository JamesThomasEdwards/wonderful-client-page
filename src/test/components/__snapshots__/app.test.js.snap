// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App component correctly 1`] = `
<div>
  <TopMiddleShapeContainer />
  <AboutProfileCardContainer />
  <BottomLeftShapeContainer />
</div>
`;
